group-coordinator/src/main/java/org/apache/kafka/coordinator/group/metrics/GroupCoordinatorRuntimeMetrics.java

```diff
@@ -42,6 +48,16 @@ public class GroupCoordinatorRuntimeMetrics implements CoordinatorRuntimeMetrics
      */
     public static final String NUM_PARTITIONS_METRIC_NAME = "num-partitions";
 
+    /**
+     * The event purgatory time metric name.
+     */
+    public static final String EVENT_PURGATORY_TIME_METRIC_NAME = "event-purgatory-time-ms";
+
+    /**
+     * The flush time metric name.
+     */
+    public static final String FLUSH_TIME_METRIC_NAME = "flush-time-ms";
```

Comment:
nit: I wonder if we should call it `batch-flush-time-ms` to make it clear that it is the flush time of batches.